.github/workflows/check-pr-title.yml

```diff
@@ -10,23 +10,13 @@ on:
 jobs:
   check-pr-title:
     runs-on: ubuntu-latest
-    permissions:
-      checks: write
-      pull-requests: read
-      contents: read
+    timeout-minutes: 5
     steps:
-      - uses: amannn/action-semantic-pull-request@v4
+      - name: Check out branch
+        uses: actions/checkout@v3
+
+      - name: Validate PR title
+        id: validate_pr_title
+        uses: ivov/validate-n8n-pull-request-title@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
do we still need the `GITHUB_TOKEN` for anything?

Reply:
For [API request](https://github.com/ivov/validate-n8n-pull-request-title/blob/a3616cc20f5a2bc6e6180270cac95253c1651836/index.js#L19-L21), because PR data in context is outdated on title update.